airbrakes/airbrakes.py

```diff
@@ -61,7 +61,7 @@ def __init__(
         self.apogee_predictor = apogee_predictor
 
         # Placeholder for the current airbrake extension until they are set
-        self.current_extension: ServoExtension = ServoExtension.MIN_EXTENSION
+        self.current_extension: ServoExtension = ServoExtension.MIN_EXTENSION.value
```

Comment:
We should be leaving this as the enum rather than getting its value. If for some reason you want extensions to be represented as floats rather than as enums that you unpack, then we need to change all of the types back to floats.